packages/rest/src/router/base-route.ts

```diff
@@ -36,18 +36,18 @@ export abstract class BaseRoute implements RouteEntry {
   ): Promise<OperationRetval>;
 
   describe(): string {
-    return `"${this.verb} ${this.path}"`;
+    return `${this.verb} ${this.path}`;
```

Comment:
Please preserve double quotes, they help to troubleshoot as silly edge case where the path contains a space (I have experienced this in the past).

Reply:
IMO, it's better to quote it by consumers of `describe()`. For example, if the subclass wants to override `describe` as follows:

```ts
describe() {
  const des = super.describe(); 
  return `... ${des} ...`; // it's much harder to remove quotes than adding new ones
}
```